pegasus/sites.v3/code.org/public/educate/district/partners.md

```diff
@@ -1,6 +1,7 @@
 ---
```

Comment:
map on this page needs to scale and we'll have to test how the scrolling/zooming works